### DISCUSSION TITLE
Teacher Admin: Demo changes

### DISCUSSION THREAD

**Discussion by fakhruddin-vsbizz** [priyadarshi0811/arabic-moallema-refactored-v2.0#58] [Announcements]
Demo changes for Teacher admin

**Comment by fakhruddin-vsbizz** [comment]
Teacher: Batch Progress Detail - Remove checkbox and icon in present absent list

**Reply by atif-vsbizz** [reply]
[Done]


![2023-03-17_153538](https://user-images.githubusercontent.com/124858584/225875329-e36e3d39-2aec-4b5f-be4f-7f580fb0d419.png)

**Reply by fakhruddin-vsbizz** [reply]
Nice work.

**Comment by fakhruddin-vsbizz** [comment]
Teacher: Student report - Remove the unchecked assignments section.

**Reply by somesh2001** [reply]
Removed the section for unchecked assignment

![image](https://user-images.githubusercontent.com/56119880/225863115-dc306448-162e-4826-a9e9-fa3e8f464fc1.png)

**Comment by fakhruddin-vsbizz** [comment]
Teacher: Student assignment marking - When marked and viewed, Grade Assignment button should be removed.

**Reply by somesh2001** [reply]
Done - When marked and viewed, the Grade Assignment button is removed

![image](https://user-images.githubusercontent.com/56119880/225861947-0ebc15af-495c-4647-a14c-3a27b7ab1bde.png)

**Comment by fakhruddin-vsbizz** [comment]
Teacher: Start class - Label instead of Textbox beside Join Meet button

**Reply by somesh2001** [reply]
Done

![Screenshot (466)](https://user-images.githubusercontent.com/56119880/225860353-22ea354b-0589-490f-bb93-acb558663408.png)

**Reply by fakhruddin-vsbizz** [reply]
Good. I just to ask since I am getting confused, is the link in button or is it a simple label

**Reply by somesh2001** [reply]
Update Label For Link

![image](https://user-images.githubusercontent.com/56119880/225877168-ab603694-7965-43de-ab58-851bf0d07b98.png)

**Reply by fakhruddin-vsbizz** [reply]
Okay good.
